benchmarks/run_stardis.py

```diff
@@ -0,0 +1,23 @@
+# Import necessary code
+
+import os
+import numpy as np
+from tardis.io.atom_data.util import download_atom_data
+from stardis.base import run_stardis
+from astropy import units as u, constants as const
+
+
+class Benchmarkrunstardis:
+    """
+    Class to benchmark run_stardis functions.
+    """
+
+    timeout = 1800  # Worst case timeout of 30 mins
+
+    def setup(self):
+        self.curr_dir = os.path.dirname(os.path.realpath(__file__))
+
+    def time_run_tardis(self):
```

Comment:
This looks like a typo - should be time_run_stardis()

Reply:
Yes thanks